packages/wrangler/src/dev/miniflare.ts

```diff
@@ -322,6 +356,7 @@ export function buildMiniflareBindingOptions(config: MiniflareBindingsConfig): {
 		// script. They don't need to persist anything, and would end up using the
 		// incorrect unsafe unique key.
 		unsafeEphemeralDurableObjects: true,
+		compatibilityDate: "2024-01-01",
```

Comment:
Been a hot minute since I was in here. This is a system Worker we spin up to proxy the DOs, right? So just double-checking that we're good to set our own compat date here? We don't need to use a user-provided value of either the caller or callee, right?